packages/flutter_test/lib/src/binding.dart

```diff
@@ -1180,7 +1180,7 @@ abstract class TestWidgetsFlutterBinding extends BindingBase
 }
 
 /// A variant of [TestWidgetsFlutterBinding] for executing tests in
-/// the `flutter test` environment.
+/// the host environment.
```

Comment:
I don't think most people will understand what is meant by "host environment". We should be more specific here. It should probably continue to mention that this binding is _typically_ used when running `flutter test` - unless it is an integration test.

Reply:
done